### PR TITLE
INDY-2176:  indy-node broken by indy-plenum and python-dateutil 

### PR DESCRIPTION
**Pull request by NeolithEra**
Signed-off-by: NeolithEra <3226592650@qq.com>

### CODE REVIEW

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

**Comment by NeolithEra** [comment]
https://jira.hyperledger.org/browse/INDY-2176

**Comment by NeolithEra** [comment]
Hi, users are unable to run _**indy-node **_ due to dependency conflict with _**python-dateutil**_ package.
As shown in the following full dependency graph of indy-node, indy-node requires _**python-dateutil**_ (the latest version), while _**indy-plenum==1.9.0.dev847**_ requires _**python-dateutil==2.6.1**.
According to pip’s “first found wins” installation strategy, _**python-dateutil==2.8.0**_ is the actually installed version. However, _**python-dateutil==2.8.0**_ does not satisfy _**==2.6.1**_.

### Dependency tree-------
```
indy-node<version range:>
| +-distro<version range:==1.3.0>
| +-indy-plenum<version range:==1.8.1>
| | +-base58<version range:==1.0.0>
| | +-indy-crypto<version range:==0.4.5>
| | | +-pytest<version range:>
| | +-intervaltree<version range:==2.1.0>
| | | +-sortedcontainers<version range:>
| | +-ioflo<version range:==1.5.4>
| | +-jsonpickle<version range:==0.9.6>
| | +-leveldb<version range:>
| | +-libnacl<version range:==1.6.1>
| | +-msgpack-python<version range:==0.4.6>
| | +-orderedset<version range:==2.0>
| | +-packaging<version range:==19.0>
| | +-pip<version range:<10.0.0>
| | +-portalocker<version range:==0.5.7>
| | +-prompt-toolkit<version range:==0.57>
| | +-psutil<version range:==5.4.3>
| | +-pygments<version range:==2.2.0>
| | +-pympler<version range:==0.5>
| | +-python-dateutil<version range:==2.6.1>
| | +-python-rocksdb<version range:==0.6.9>
| | | +-setuptools<version range:>=25>
| | +-pyzmq<version range:==17.0.0>
| | +-rlp<version range:==0.5.1>
| | +-semver<version range:==2.7.9>
| | +-sha3<version range:==0.2.1>
| | +-six<version range:==1.11.0>
| | +-sortedcontainers<version range:==1.5.7>
| | +-ujson<version range:==1.33>
| +-python-dateutil<version range:>
| +-timeout-decorator<version range:==0.4.0>

```

### Solution
Fix version of python-dateutil as ==2.6.1 to solve this conflict.

Thanks for your help.
Best,
Neolith

**Comment by NeolithEra** [comment]
@andkononykhin, Hi, could you help me review this issue?

File: setup.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
         (BASE_DIR, ['data/nssm_original.exe'])
     )],
     install_requires=['indy-plenum==1.9.0.dev847',
-                      'python-dateutil',
+                      'python-dateutil==2.6.1',
```
Comment (ashcherbakov) [inline setup.py]: I think this dependency can be removed at all since plenum already depends on this version of python-dateutil

**Comment by ashcherbakov** [comment]
@NeolithEra 
Thanks for fixing this!
In order to merge the PR, can you please do the following:
1) Add a DCO signature to your commits
2) Resolve conflict in setup.py

**Comment by NeolithEra** [comment]
> @NeolithEra
> Thanks for fixing this!
> In order to merge the PR, can you please do the following:
> 
> 1. Add a DCO signature to your commits
> 2. Resolve conflict in setup.py

Sorry，what does DCO mean?

**Comment by ashcherbakov** [comment]
@NeolithEra This is Developer Certificate of Origin (https://github.com/apps/dco).
Please have a look at https://github.com/hyperledger/indy-node/pull/1389/checks?check_run_id=175180131. It has the commands for how to add it for your commits.

**Comment by esplinr** [comment]
@NeolithEra If you need help fixing the DCO, we can schedule a call and walk you through the process. We appreciate you contributing to the project!

**Comment by ashcherbakov** [comment]
Thanks for finding the issue and your contribution. The fix was done in https://github.com/hyperledger/indy-node/pull/1411

**Pull request closed without merging**